odl/contrib/mrc/README.md

```diff
@@ -0,0 +1,26 @@
+# MRC
```

Comment:
Well, as I wrote somewhere else, MRC doesn't provide enough information in itself to create a geometry, so there's not really a good way to get there in a front-page example. People using MRC know that from own painful experience :-)